packages/static/grub-config/files/grub.cfg

```diff
@@ -54,6 +54,7 @@ insmod all_video
 insmod gfxterm
 insmod loopback
 insmod squash4
+terminal_output gfxterm
```

Comment:
I wonder if it's possible that we break the serial console with this setting (https://docs.fedoraproject.org/en-US/quick-docs/bootloading-with-grub2/#using-old-graphics-modes). I don't understand the implications of changing this value.

Reply:
good question, I'll test it out to be sure it doesn't

Reply:
very nice catch @jimmykarily it was failing indeed, but I managed to find a solution, will merge and try everything with the osbuilder